freegames/madlibs.py

```diff
@@ -0,0 +1,59 @@
+""" Mad Libs: Funny Story Creation Game.
+
+Exercises:
+
+1. Can you create functionality to replace the given story?
+2. Can you load the story and template from a file?
+3. Can you add additional tenses?
+4. Can you add flexibility in a loaded story template?
+5. Can you adjust for grammar? (past tense, etc.)
+
+"""
+
+story = "The [adj] [adj] [n] [v] over the [adj] [adj] [n]"
+# The quick black fox jumped over the lazy brown dog.
+
+word_type_counter = {}
+
+word_storage = {}
+
+word_type_key = {
+    "adj": "adjective",
+    "n": "noun",
+    "v": "verb",
+    "adv": "adverb"
+}
+
+def extract_word_types():
+    "Word types required extracted from story"
+    for w_type in word_type_key:
+        lookup = "[{}]".format(w_type)
```

Comment:
Remove `lookup` by making the keys "[adj]" rather than "adj" in `word_type_key`.

Reply:
Great suggestion. Made the program much simpler. Perfect for the purpose of this repository!